ios/TransactionListView.swift

```diff
@@ -50,7 +50,14 @@ class TransactionListView: UIView, UITableViewDelegate {
     }
   }
   @objc func onAvatarPressed(_ sender: UIButton) {
-    self.onAvatarPress([:])
+    tableView.setContentOffset(.zero, animated: true)
+    if tableView.contentOffset.y == CGFloat(0) {
+      self.onAvatarPress([:])
+    } else {
+      DispatchQueue.main.asyncAfter(deadline: .now() + 0.2) {
```

Comment:
Can you move the 0.2 to a var at the top specifying what the delay is accounting for?
```
@objc var delayForSomething: TimeInterval = 0.2
```

Reply:
Resolved 🎉
Sorry, It took me so long I was sure I committed it already, but I actually must have stashed it somewhere along the way :/